### PR TITLE
Update logo Sass option documentation

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ You can be more selective about which types you would like to output, by using a
 - `'b2c'`
 
 **logo**
-- The name of a logo from the [logos image set](https://registry.origami.ft.com/components/logo-images@1.8.0). Defaults to the FT logo.
+- The logo image to show in the navigation. The image must come from an Origami [image set](https://registry.origami.ft.com/components?imageset=true&active=true&maintained=true). Specify the image set and image name as a string `[imageset]:[image]` e.g `ftlogo-v1:origami`. Defaults to the FT logo.
```
Comment (notlee) [inline README.md:116]:
```suggestion
- The logo image to show in the header. The image must come from an Origami [image set](https://registry.origami.ft.com/components?imageset=true&active=true&maintained=true). Specify the image set and image name as a string `[imageset]:[image]` e.g `ftlogo-v1:origami`. Defaults to the FT logo.
```

**Comment by origamiserviceuser** [comment]
o-header-services bundle size difference from 4.0.6 to e56da46d9d73803a90a5c70df21037a935622da1
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-header-services bundle size difference from 4.0.6 to 62e2cd80696171150c5990f4a05f913c2dd71dd2
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v4.0.7 :tada:

**Pull request merged**